app.py

```diff
@@ -0,0 +1,8 @@
+from aws_lambda_powertools import Logger
+
+def handler(event, context):
+    for record in event['records']:
+        payload = record["body"]
+
+        # this should log the event
+        logger = Logger(service=payload)
```

Comment:
This shouldn't be in the handler. I think there's a misunderstanding of how the logger works. That line initialized the logger and creates the log group. The service parameter is the name of your service. 

Take a look at this https://awslabs.github.io/aws-lambda-powertools-python/latest/core/logger/#capturing-lambda-context-info

Reply:
So I refactored my handler function using a logger decorator above the function and attempt to log the payload